test/exprtest/CMakeLists.txt

```diff
@@ -19,41 +19,37 @@
 # *
 # *  SYCL-BLAS: BLAS implementation using SYCL
 # *
-# *  @filename executor_base.hpp
+# *  @filename CMakeLists.txt
 # *
 # **************************************************************************/
 cmake_minimum_required(VERSION 3.2.2)
 
-project(syclblas_unittest)
+project(syclblas_exprtest)
 
 set(SYCLBLAS_EXPRTEST ${CMAKE_CURRENT_SOURCE_DIR})
 
-include_directories(${SYCLBLAS_TEST})
-
 # compiling tests
-file(GLOB SYCL_EXPRTEST_SRCS
-  ${SYCLBLAS_EXPRTEST}/blas1_iface_test.cpp
+set(SYCL_EXPRTEST_SRCS
+  ${SYCLBLAS_EXPRTEST}/blas1_axpy_copy_test.cpp
 )
 
+include_directories(${SYCLBLAS_TEST} ${SYCLBLAS_SRC} ${SYCLBLAS_INCLUDE} ${THIRD_PARTIES_INCLUDE})
```

Comment:
I think `target_include_directories` is preferred as we use CMake > 3. It's scope is restricted to a target instead of being applied to all targets created from this directory. @mmha, any opinion on this?

Reply:
I think all targets created from this directory will have the same include_directories. But I can change this.